### PR TITLE
Use `prettier` to format files

### PR DESCRIPTION
**Pull request by EyeOfPython**
Remove "standard" eslints in favor of "prettier", which fixes stuff like line lengths.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,11 +82,10 @@
     "@types/vcf": "^2.0.3",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
-    "@vue/devtools": "^5.3.4",
```
Comment (schancel) [inline package.json:84]: Why's this being removed?

File: src/cashweb/auth_wrapper/wrapper_pb.d.ts [hunk 2]
```diff
@@ -1,114 +1,148 @@
 // package: wrapper
 // file: wrapper.proto
 
-import * as jspb from "google-protobuf";
+import * as jspb from 'google-protobuf'
```
Comment (schancel) [inline src/cashweb/auth_wrapper/wrapper_pb.d.ts:4]: These should have been excluded from the linter entirely as they're generated files.

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+node_modules
+local_modules
+dist
```
Comment (schancel) [inline .prettierignore:3]: Need to add the protobufs stuff here.

**Review by schancel — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (EyeOfPython) [reply package.json:84]: Breaks yarn completely.

Remove as discussed.

**Pull request merged**